packages/xrpl/src/client/partialPayment.ts

```diff
@@ -159,10 +160,10 @@ export function handlePartialPayment<
  * @param log - The method used for logging by the connection (to report the partial payment).
  */
 export function handleStreamPartialPayment(
-  stream: TransactionStream,
+  stream: TransactionStream | TransactionV1Stream,
   log: (id: string, message: string) => void,
 ): void {
-  if (isPartialPayment(stream.transaction, stream.meta)) {
+  if (isPartialPayment(stream.transaction ?? stream.tx_json, stream.meta)) {
```

Comment:
I'd prefer it to be: 
```suggestion
  if (isPartialPayment(stream.tx_json ?? stream.transaction, stream.meta)) {
```

In the future rippled API versions, even if stream.transaction is deprecated/removed, this javascript code will still work correctly